src/components/Root.js

```diff
@@ -1,20 +1,22 @@
 import React, { Component, PropTypes } from 'react';
 import { Provider } from 'react-redux';
-import routes from '../routes';
-import { Router } from 'react-router';
+import { ConnectedRouter } from 'react-router-redux';
+import App from './App';
 
 export default class Root extends Component {
   render() {
     const { store, history } = this.props;
     return (
       <Provider store={store}>
-        <Router history={history} routes={routes} />
+        <ConnectedRouter history={history}>
+          <App />
+        </ConnectedRouter>
       </Provider>
     );
   }
 }
 
 Root.propTypes = {
   store: PropTypes.object.isRequired,
-  history: PropTypes.object.isRequired
+  history: PropTypes.object.isRequired,
```

Comment:
@oshalygin lol, i seem to be the only one who hasn't disabled that rule in .eslintrc files.  Newest commit has both changes you suggested. Thanks for the feedback!